### ISSUE TITLE
Make color picker keyboard accessible

### ISSUE THREAD

**Issue by jodator** [ckeditor/ckeditor5#6239]
## 📝 Provide a description of the improvement

You cannot open the color picker from the table / table cell properties form using the keyboard.

## 📃 Other details

Linked issues:
- [color dropdown keyboard navigation enhancement](https://github.com/ckeditor/ckeditor5/issues/2285)

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.

**Comment by Dumluregn** [comment]
This is partially improved at the branch `ck/color-picker-focus-fix` - after choosing a color from picker, focus is moved to the color input. However we aim at the holistic solution, i.e. making these color pickers focusable, so it can be just an inspiration how to move the focus after all.

**Comment by mlewand** [comment]
We fixed it while working on color picker a little while ago. Fixed either in v39 or v38.
